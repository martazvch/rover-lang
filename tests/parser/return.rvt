// Tests return statement

code:
return
return 56

expect:
[Return]
[Return
    [Int literal 56]
]

================

code:
return return

error:
ExpectExpr, return

================

