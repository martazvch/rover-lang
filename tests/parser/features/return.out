[@"return"]
[@"return"
    [Int literal 56]
]
